cmd/tailscale/cli/up.go

```diff
@@ -221,9 +224,17 @@ func runUp(ctx context.Context, args []string) error {
 			if n.ErrMessage != nil {
 				fatalf("backend error: %v\n", *n.ErrMessage)
 			}
-			if s := n.State; s != nil {
+			if upArgs.forceReauth && !loginStarted {
+				loginStarted = true
+				printed = true
+				bc.StartLoginInteractive()
```

Comment:
Why does this need to be in response to a Notify? Why not just down on line 264 before or after or instead of the existing `Start` call?